src/example.config.json

```diff
@@ -0,0 +1,8 @@
+{
+	"port": 8080,
+	"bodyLimit": "100kb",
+	"corsHeaders": ["Link"],
+    "mysql_user": "?????",
```

Comment:
Now that we're using the `.env` instead of the config you can just delete this file